backend/src/utils/web_utils.py

```diff
@@ -13,7 +13,7 @@
 engine = PromptEngine()
 
 
-async def search_urls(search_query, num_results=10) -> str:
+async def search_urls(search_query, num_results=30) -> str:
```

Comment:
Should this be part of this PR?

Reply:
@IMladjenovic - This MR is on top of the Web agent one...and the web agent one is yet to me merged back to the main..
How should I handle it?